modules/users/client/components/AboutMe.component.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { NamespacesConsumer } from 'react-i18next';
+import '@/config/lib/i18n';
+import PropTypes from 'prop-types';
+import { limitTo, plainTextLength } from './utils/filters';
+
+export default function AboutMe({ profile, profileDescriptionToggle, user, appSettings }) {
+  return (<NamespacesConsumer ns={['user-profile']}>{ (t) => (<>
+    <section className="panel panel-default">
+      <header className="panel-heading">
+        {t('About me')}
+      </header>
+      <div className="panel-body">
+        {/* Short descriptions */}
+        {(profile.description.length < 2400 || profileDescriptionToggle) &&
+        <div dangerouslySetInnerHTML={{ __html: profile.description }} ></div>}{/* TODO Can we trust this HTML?*/}
```

Comment:
@simison Can we trust `profile.description`? Is it well sanitized when it's saved to database? Or should we worry about [XSS](https://www.owasp.org/index.php/Cross-site_Scripting_(XSS)) here?

Reply:
We sanitize it on save _and_ output (most importantly):

https://github.com/Trustroots/trustroots/blob/65d2ad2bf7a2aafdc439c6b11994ee1278098501/modules/users/server/controllers/users.profile.server.controller.js#L906-L907

I'd say we can trust that for now. Other fields are not to be trusted.

Note also that _if_ we do any frontend sanitization here, we should pass `textService.sanitizeOptions` to `sanitize-html` so that we sanitize with same settings at the back- end frontend.

Reply:
I removed sanitizing from the frontend.